src/scripts/branch_review.py

```diff
@@ -0,0 +1,295 @@
+            OI.relationships(
+                subjects=children_of_obsoletion_candidate,
+                predicates=[IS_A, PART_OF],
+            )
+        )
+
+        children_also_obsoletion_candidates = children_of_obsoletion_candidate & obsoletion_candidate_child_of_branch
+
+        for child in children_of_obsoletion_candidate:
+            # Column B: Children of ObsoleteCandidate == child
+            parents_of_child = set(
+                get_immediate_parent(
+                    curie=child,
+                    relationships=children_of_obsoletion_candidate_parents_relationship,
+                )
+            ) - all_obsoletes
+
+            # These are parents that need to be obsoleted and hence NOT in Branch
+            relevant_obsoletion_parents_in_branch = (
+                parents_of_child & obsoletion_candidate_child_of_branch
+            )
+
+            parents_of_child.difference_update(obsoletion_candidate_child_of_branch)
+
+            # Column F: Other direct parents in Branch = other_parents_in_branch
+            other_parents_in_branch = parents_of_child & branch_descendants_set
+
+            # Column G: Other direct parents NOT in Branch = other_parents_not_in_branch
+            other_parents_not_in_branch = (
+                parents_of_child - other_parents_in_branch
+            )
+
+            all_ancestors = (
+                set(OI.ancestors([child], predicates=[IS_A, PART_OF], method="ENTAILMENT"))
+                - parents_of_child
+                - {child}
+            )
+            all_ancestors = {
+                anc for anc in all_ancestors if str(anc).startswith("MONDO:")
+            }
+
+            all_ancestors.difference_update(obsoletion_candidate_child_of_branch)
+
+            # Column H: Ancestor inside the branch = all_mondo_ancestors_in_branch
+            all_ancestors_in_branch = (
+                all_ancestors & branch_descendants_set
+            )
+
+            # Column I: Ancestor outside the branch = all_mondo_ancestors_outside_branch
+            all_ancestors_outside_branch = (
+                all_ancestors - all_ancestors_in_branch
+            )
+
+            # if child == "MONDO:0013310":
+            #     import pdb; pdb.set_trace()
+
+            other_parents_in_branch = add_obsoleted_label(other_parents_in_branch, obsoletion_candidates_set)
+            other_parents_not_in_branch = add_obsoleted_label(other_parents_not_in_branch, obsoletion_candidates_set)
+            all_ancestors_in_branch = add_obsoleted_label(all_ancestors_in_branch, obsoletion_candidates_set)
+            all_ancestors_outside_branch = add_obsoleted_label(all_ancestors_outside_branch, obsoletion_candidates_set)
+            relevant_obsoletion_parents_in_branch = add_obsoleted_label(relevant_obsoletion_parents_in_branch, obsoletion_candidates_set)
+
+            # Column J: Affected Status = status
+            status = ""
+            if len(other_parents_in_branch) > 0 and all(
```

Comment:
```suggestion
            if len(other_parents_in_branch) > 0 and any(
```

This seems to be the last issue in the table!